packages/app/components/Settings/EditStudentModal.tsx

```diff
@@ -0,0 +1,108 @@
+import { ReactElement } from "react";
+import Modal from "antd/lib/modal/Modal";
+import { Input, Form } from "antd";
+import { API } from "@koh/api-client";
+import { default as React } from "react";
+import { mutate } from "swr";
+import { useState, useEffect } from "react";
+import { Role, UBCOuserParam } from "@koh/common";
+
+interface EditStudentModalProps {
+  courseId: number;
+  student: any;
+  onClose: () => void;
+}
+
+export function EditStudentModal({
+  courseId,
+  student,
+  onClose,
+}: EditStudentModalProps): ReactElement {
+  const [form] = Form.useForm();
+  const [page] = useState(1);
+  const [search] = useState("");
+
+  const editStudent = async (userData: UBCOuserParam, id: number) => {
+    console.log(userData);
+    console.log(id);
+    await API.profile.editUser(userData, id);
```

Comment:
surround these in try catch if possible, the API calls can go wrong and we want the user to understand what's going on instead of crash